UI/Controllers/PlayerController.cs

```diff
@@ -26,8 +28,11 @@ public async Task<IActionResult> GetAll()
         {
             return NotFound();
         }
+        
+        var playersList = players.ToList();
+        var playersListRequest = Mapper.MapPlayersToPlayerRequests(playersList);
 
-        return Ok(players);
+        return Ok(playersListRequest);
```

Comment:
Why is it playerList -> REQUEST? It's a response! And would make more sense to be `playerListResponse`

Reply:
fixed

Reply:
fixed